draft/accessibility-team.rst

```diff
@@ -0,0 +1,129 @@
+
+- Have disabilities that make using the web and/or web development more
+  difficult.
+- Have relevant, positive contributions or expertise in the field.
+- Have a record of contributing to Django.
+
+Members shall remain in the team for a fixed-term of 9 months, after which
+they must opt-in to remain on the team. Membership will also be terminated by:
+
+- Becoming disqualified due to actions taken by the Code of Conduct committe
+  of the Django Software Foundation.
+
+- A vote of the Technical Board, or full consensus of the rest of the
+  Accessibility Team, if the team is considered too large, the person is not
+  making positive contributions, or any other sound reason.
+  (TODO: dunno about any of this)
+
+Responsibilities
+----------------
+
+The remit of the Accessibility Team shall include all user-facing components
+of projects maintained by the Django Software Foundation. This includes but is
+not limited to:
+
+- User-facing parts of Django, such as default HTML output in forms and such,
+  and the django admin and its default theme, HTML, and JavaScript components.
+
+- All websites and other software projects maintained by the DSF, such as the
+  Django website, including documentation, Django Snippets, Django People,
+  the issue tracker, etc.
+
+The responsibilities of the Accessibility Team are expected to change over
+time, and are to be decided by conensus of the team, with input from the
+Technical Board as required. to begin, several areas have been identified:
+
+- Deciding on any relevant accessibility guidelines to follow, such as WCAG,
+  and at which conformance level.
+
+- Implementing automated testing to catch simpler issues, working with the ops
+  team as needed to integrate this into the CI process.
+
+- Fixing accessibility issues in Django and associated projects.
+
+- Reviewing accessibility fixes and other tickets that may affect the
+  accessibility of the project.
+
+To aid in reviewing, the Accessibility Team members shall be added to a team
+in the GitHub organization with read access to relevant repositories, so that
+they may be requested to review pull requests, at the discretion of the author,
+reviewer, or other party.
+
+Many of these duties can be implemented by any contributor, not only by the
+Accessibility Team, however the Accessibility Team exists here as a fallback
+in case nobody else is available to do so.
+
+
+Motivation
+==========
+
+Accessible websites are not technically difficult to create, but accessibility
+is often overlooked when developing features. Django is no exception to this.
+The django admin currently exhibits many accessibility issues, such as forms
+fields missing labels, low color contrast, a lacklustre keyboard-only
+experience among other issues. Problems exist on other sites maintained by the
+DSF, such as the `Django website <https://www.djangoproject.com/>`__,
+`Django Snippets <https://djangosnippets.org/>`_, and others.
```

Comment:
I think this should also state that there might be problems with Django’s features, for example `as_table` for forms.

Reply:
This is addressed earlier, "User-facing parts of Django, such as default HTML output in forms and such". I'm not fully aware of any a11y issues with it. If you know of any, let me know and I can add them.

Reply:
👍 for `as_table`, it’s simply inappropriate to use tables for layout of form fields like this, or for any kind of layout really. Tables should be for data only.